modules/ossm-installing-external-control-plane.adoc

```diff
@@ -0,0 +1,457 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-external-control-plane_{context}"]
+= Installing the control plane and data plane on separate clusters
+
+Install {istio} on a control plane cluster and a separate data plane cluster. This installation approach provides increased security.
+
+[NOTE]
+====
+You can adapt these instructions for a mesh spanning more than one data plane cluster. You can also adapt these instructions for multiple meshes with multiple control planes on the same control plane cluster.
+====
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on the control plane cluster and the data plane cluster.
+
+* You have `istioctl` installed on the laptop you will use to run these instructions.
+
+.Procedure
+
+. Create an `ISTIO_VERSION` environment variable that defines the {istio} version to install on all the clusters by running the following command:
++
+[source,terminal]
+----
+$ export ISTIO_VERSION=1.24.1
+----
+
+. Create a `REMOTE_CLUSTER_NAME` environment variable that defines the name of the cluster by running the following command:
++
+[source,terminal]
+----
+$ export REMOTE_CLUSTER_NAME=cluster1
```

Comment:
@luksa : We are not using this "REMOTE_CLUSTER_NAME" any where, can we remove from the doc ?